patches/tModLoader/Terraria/ModLoader/IO/TagSerializer.cs

```diff
@@ -71,6 +71,13 @@ public abstract class TagSerializer
 
 			return null;
 		}
+
+		public void Load(Mod mod) {
+			AddSerializer(this);
```

Comment:
AddSerializer should be made internal, since you made it be superseded by mod.AddContent.